Campus-iOS/Map/MainView/Panel.swift

```diff
@@ -26,12 +24,16 @@ struct Panel: View {
             .onEnded(onDragEnded)
         
         return Group {
-            PanelContent(zoomOnUser: $zoomOnUser,
+            VStack{
+                PanelContent(zoomOnUser: $zoomOnUser,
                          panelPosition: $panelPosition,
                          canteens: $canteens,
-                         selectedCanteenName: $selectedCanteenName,
-                         selectedAnnotationIndex: $selectedAnnotationIndex,
                          selectedCanteen: $selectedCanteen)
+                
+                //TODO: check for better way, to make last items in list available
```

Comment:
Left todo?

Reply:
There was a bug, that the last item in the panel was not properly shown. 
I fixed this, by adding empty space to the panel (which is then hidden behind the navigation bar).

I tried for several hours, to change the swiping behavior of the panel to be correct, but didn't manage to get it better.
So the current version kind of works, but is not ideal. However, I don't know how to make it better.